CodingStandard/Tests/Classes/ClassCreateInstanceUnitTest.php

```diff
@@ -43,6 +43,10 @@ public function getErrorList($testFile)
                 // Space after class name and it's opening brace.
                 4 => 1,
                 5 => 1,
+                7 => 1,
```

Comment:
After performing changes requested above the error lines would be grouped correctly under comments. Right now error in line 7 for example doesn't match `// Space after class name and it's opening brace.` comment.